src/Quotas/QuotasProvider.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+namespace Sherl\Sdk\Quota;
+
+use Exception;
+use GuzzleHttp\Client;
+use Sherl\Sdk\Common\SerializerFactory;
+use Sherl\Sdk\Quotas\Dto\QuotaOutputDto;
+use Sherl\Sdk\Common\Error\SherlException;
+use Sherl\Sdk\Common\Error\ErrorFactory;
+use Sherl\Sdk\Common\Error\ErrorHelper;
+use Sherl\Sdk\Quotas\Errors\QuotasErr;
+use Sherl\Sdk\Quotas\Dto\QuotaFilterDto;
+use GuzzleHttp\RequestOptions;
+
+class QuotaProvider
+{
+    public const DOMAIN = 'Quota';
+
+    private Client $client;
+    private ErrorFactory $errorFactory;
+    /**
+     * Constructor for QuotaProvider.
+     * @param Client $client The HTTP client used to make requests.
+     */
+    public function __construct(Client $client)
+    {
+        $this->client = $client;
+        $this->errorFactory = new ErrorFactory(self::DOMAIN, QuotasErr::$errors);
+    }
+
+    /**
+     * Retrieves quota information based on provided filters.
+     * @param QuotaFilterDto $filters The filters to apply when retrieving quota data.
+     * @return QuotaOutputDto|null The quota data transfer object if found, null otherwise.
+     * @throws SherlException If the response status code indicates an error.
+     */
+    public function getQuotaFindOneBy(QuotaFilterDto $filters): ?QuotaOutputDto
+    {
+        try {
+            $response = $this->client->post("/api/quotas/findOneBy", [
+                "headers" => [
+                "Content-Type" => "application/json",
+                ],
+                RequestOptions::JSON => [
```

Comment:
À vérifier mais j'imagine que ce serait plutôt quelque chose comme ça puisqu'il n'y a pas de propriété "filters" côté API.

```suggestion
                RequestOptions::QUERY => $filters
```